docs/integration.md

```diff
@@ -608,25 +615,147 @@ logIndex | Index of the burn event in the transaction receipt
+The transactionID is a unique identifier for the Bridge transfer operation and can be used to query the status of the Bridge transfer.
+
+### Step 2. Waiting for Signatures
+
+After the user has submitted the deposit transfer, he can begin to query a validator's API to see whether the necessary components for the transfer are met:
+
+    GET {validator_url}/api/v1/transfers/{transaction_id}
+
+Where `transaction_id` is the Hedera `TransactionID` of the NFT transfer to the Bridge account.
+
+The response is in JSON format and contains the following data:
+
+```json
+{
+  "isNft": true,
+  "recipient": "0x700d8a76b37f672a06ab89fe1ec95acfba799f1c",
+  "routerAddress": "0x",
+  "sourceChainId": "",
+  "targetChainId": "",
+  "sourceAsset": "",
+  "nativeAsset": "",
+  "targetAsset": "",
+  "tokenId": "",
+  "metadata": "",
+  "signatures": [
+  ],
+  "majority": false
+}
+```
+
+Property | Description
+---------- | ----------
+**isNft** | Whether the transfer is fungible or non-fungible
+**recipient** | EVM address of the receiver
+**routerAddress** | Address of the router contract
+**sourceChainId** | The chain ID from which the transfer has been initiated
+**targetChainId** | The chain ID to which the transfer data and has to be submitted
+**sourceAsset** | The asset from the source chain
+**nativeAsset** | The native asset of the transferred asset
+**targetAsset** | The target asset for the transfer
+**tokenId** | The tokenId of the NFT
+**metadata** | The metadata/tokenURI of the NFT
+**signatures** | Array of all provided signatures by the validators
+**majority** | True if supermajority is reached and the wrapped token may be claimed
+
+### Step 3. Mint Wrapped Asset
+
+Once majority is reached, users can mint the _wrapped version_ of the NFT. In order to do that, users must submit a `mintERC721` transaction to the Bridge Router Contract.
+
+The mint operation can be constructed using the following arguments:
+
+	mintERC721(uint256 sourceChainId, bytes transactionId, address targetAsset, uint256 tokenId, string metadata, address recipient, bytes[] signatures)
+
+Argument | Description
+---------- | ----------
+**transactionId** | The Hedera `TransactionID` of the Deposit transaction. Converting the TX ID string to bytes for JS/TS: `Web3.utils.fromAscii(transactionId)` or `ethers.utils.toUtf8Bytes(transactionId)`.
+**signatures** | Depending on the library chosen for EVM submission, it might be required to add to each signature a `0x` prefix.
+
+## NFT Transfers from EVM back to Hedera
+The steps below will showcase a bridge transfer of a wrapped NFT from any EVM back to Hedera.
+
+### Step 1. Burn the Wrapped NFT Asset
+Wrapped NFT burn consists of the following actions:
+* An ERC-20 fee for the ERC-721 is sent from the user to the Router contract.
+* An ERC-721 tokenId is burnt from the user.
+
+Before user submits the burn transaction, two approvals will be needed:
```

Comment:
I think that we should mention that the receiver of the nft should be "auto-associated" or "manually associated" to the NTF token